compat/src/suspense.js

```diff
@@ -60,6 +61,7 @@ Suspense.prototype._childDidSuspend = function(promise) {
 		if (!--c._suspensions) {
 			c._vnode._children[0] = c.state._suspended;
 			c.setState({ _suspended: (c._detachOnNextRender = null) });
+			suspender.forceUpdate();
```

Comment:
Don't we need to forceUpdate all the children that once suspended here? I.e. if `!--c._suspensions` is false that child needs to be force updated once `!--c._suspensions` goes to through in a later resolved suspension. Or am I missing a point here?

Reply:
Oh true! Lemme extend my tests and try again

Reply:
Added tests. Need to think more on this...